packages/main/src/Link.js

```diff
@@ -275,10 +276,18 @@ class Link extends UI5Element {
 	}
 
 	_onkeydown(event) {
+		if (isEnter(event)) {
```

Comment:
if it is SPACE on keydown, probably we have to handle it someway to prevent the page from scrolling, because now it is scrolling from what I see, could you check this as well.